tests/testflows/aes_encryption/tests/common.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+modes = [
```

Comment:
is it generated manually? is it possible to check that all modes are listed here?

Reply:
These modes are defined in the Software Requirements Specification (SRS). I will add the source in Markdown.